README.md

```diff
@@ -36,6 +36,15 @@ $ gulp serve
 
 More information on the gulp tasks in [this README.md](app/templates/gulp/README.md).
 
+### Configuration for development, testing and production
+
+There are different configurations available for development, testing and production:
+'dev', 'test' and 'prod'. Specify your environment by passing a parameter to gulp
+('dev' is the default):
+* 'gulp --env dev serve'
+* 'gulp --env test test'
+* 'gulp --env prod build'
```

Comment:
Just to my understanding, so now we need to specify the env in order to execute? If it's yes, we might need to adjust Bamboo.